### ISSUE TITLE
Connection to remix ide through remixd

### ISSUE THREAD

**Issue by yann300** [ethereum/remix-vscode#15]
If I use "run and deploy" in vscode and connect to remix through remixd it works.
If i close run and deploy, reactivate it, and retry to connect, it says" remixd client is already running on this port"

**Comment by bunsenstraat** [comment]
that is true. I will make it more clear in the UI what the current state is.
